specs/tooltip.md

```diff
@@ -0,0 +1,74 @@
+# Tooltip
+
+## Overview
+
+A tooltip is a popup that displays information related to an element when the element receives keyboard focus or the mouse hovers over it. It typically appears after a small delay and disappears when Escape is pressed or on mouse out.
+
+### Use Cases
+
+Tooltips are often used to supplement UI elements and provide additional information. Tooltips will often provide accessible names or add additional context to other UI components (such as icon buttons). While tooltips are natively provided to elements via the title attribute, they are notoriously difficult to style. Additionally, they are only invoked on hover and not on focus.
+
+### Non-goals
+Tooltip widgets do not receive focus. A hover that contains focusable elements can be made using a non-modal dialog (flyout).
+
+### Prior Art/Examples
+- [Lightning Design](https://www.lightningdesignsystem.com/components/tooltips/)
+- [Office Fabric](https://developer.microsoft.com/en-us/fabric#/controls/web/tooltip)
+- [Evergreen](https://evergreen.segment.com/components/tooltip/)
+- [Carbon Design](https://www.carbondesignsystem.com/components/tooltip/code/)
+- [Ant Design](https://ant.design/components/tooltip/)
+- [Atlassian](https://atlaskit.atlassian.com/packages/core/tooltip)
+- [Bootstrap](https://getbootstrap.com/docs/4.3/components/tooltips/)
+
+---
+
+### API
+
+*Component name:*
+- `fast-tooltip`
+
+*Attributes:*
+- `anchor` - the html id of the HTMLElement which triggers the tooltip. This element references the tooltip element with aria-describedby.
```

Comment:
I think we'll want to follow the same pattern here as @scomea is using with the positioner element. We'll want a property that references the associated element as well as an attribute that takes an id. The attribute would look up by id and set the property. This also allows folks to set the associated element directly in code if they have a reference (or through a templating engine that can do that).

Reply:
matched anchored region here